cpp/include/cudf/io/json.hpp

```diff
@@ -258,6 +268,14 @@ class json_reader_options {
    * @param val Boolean value to enable/disable the experimental reader
    */
   void enable_experimental(bool val) { _experimental = val; }
+
+  /**
+   * @brief Set whether the experimental reader should keep quotes of string values.
+   *
+   * @param val Boolean value to indicate whether the experimental reader should keep quotes
+   * of string values
+   */
+  void keep_quotes(bool val) { _keep_quotes = val; }
```

Comment:
Not a great name, but it's consistent 🤷 
```suggestion
  void enable_keep_quotes(bool val) { _keep_quotes = val; }
```

Reply:
Fixed 👍